packages/core/source/commands/config-forger-bip38.ts

```diff
@@ -64,38 +64,50 @@ export class Command extends Commands.Command {
 	}
 
 	private async performConfiguration(flags: Contracts.AnyObject): Promise<void> {
-		//let decodedWIF;
+		let keystore: Keystore | undefined;
 
 		await this.components.taskList([
 			{
 				task: () => {
-					if (!flags.bip39 || (!validateMnemonic(flags.bip39) && !flags.skipValidation)) {
+					if (!flags.bip39 || !validateMnemonic(flags.bip39)) {
 						throw new Error(`Failed to verify the given passphrase as BIP39 compliant.`);
 					}
 				},
 				title: "Validating passphrase is BIP39 compliant.",
 			},
 			{
-				task: () => {
-					// decodedWIF = wif.decode(Identities.WIF.fromPassphrase(flags.bip39));
+				task: async () => {
+					// use default path
+					const path: string = "m/12381/60/0/0";
+
+					await this.app.resolve(CryptoServiceProvider).register();
+
+					const keyPair = await this.app.resolve(KeyPairFactory).fromMnemonic(flags.bip39);
+
+					keystore = await Keystore.create(
+						flags.password,
+						Buffer.from(keyPair.privateKey, "hex"),
+						Buffer.from(keyPair.publicKey, "hex"),
+						path,
+					);
 				},
-				title: "Loading private key.",
+				title: "Loading keystore.",
 			},
 			{
 				task: () => {
+					if (!keystore) {
+						throw new Error("missing keystore");
+					}
+
 					const validatorsConfig = this.app.getCorePath("config", "validators.json");
 
-					const validators: Record<string, string | string[]> = readJSONSync(validatorsConfig);
-					// validators.bip38 = bip38.encrypt(
-					//     decodedWIF.privateKey,
-					//     decodedWIF.compressed,
-					//     flags.password,
-					// );
+					const validators: Record<string, any> = readJSONSync(validatorsConfig);
```

Comment:
This read can be skipped, if we are not keeping any content from existing validators.json.